friendsofsymfony/oauth-server-bundle/1.6/config/packages/fos_oauth_server.yaml

```diff
@@ -0,0 +1,48 @@
+fos_oauth_server:
+    db_driver:                  orm # Drivers available: orm, mongodb, or propel
+    client_class:               App\Entity\Client
+    access_token_class:         App\Entity\AccessToken
+    refresh_token_class:        App\Entity\RefreshToken
+    auth_code_class:            App\Entity\AuthCode
+#    model_manager_name:         ~ # change it to the name of your entity/document manager if you don't want to use the default one.
```

Comment:
We should not have config in comments like this. 
I would prefer a link to the docs instead.